### PR TITLE
[WEBSITE-606] Page titles

### PR DESCRIPTION
**Pull request by jakemulley**

### CODE REVIEW

**Review by mattrayner — CHANGES_REQUESTED** [review]
If we're using title variables like this, perhaps the H1s should just have the title echoed out to remove duplication?

File: app/views/constituencies/current.html.haml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+- title t('.title', current: t('current'), constituencies: t('constituencies_title'))
```
Comment (mattrayner) [inline app/views/constituencies/current.html.haml]: if we did something like - `page_title = title( t('.title', current: t('current'), constituencies: t('constituencies_title')) )`

Then you could do `%h1= page_title` to remove duplication

Comment (jakemulley) [reply app/views/constituencies/current.html.haml]: Thanks @mattrayner - will do.

File: Gemfile.lock [hunk 2]
```diff
@@ -1,6 +1,6 @@
 GIT
```
Comment (mattrayner) [inline Gemfile.lock:1]: Can you reset your Gemfile.lock and run `bundle update pugin`

Comment (jakemulley) [reply Gemfile.lock:1]: This is the updated version.

**Review by mattrayner — APPROVED** [review]

**Review by vinternet-pds — APPROVED** [review]

**Pull request merged**